app/tests/components/Form/ApplicationFormStatus.test.tsx

```diff
@@ -65,7 +65,7 @@ describe('The application form', () => {
     componentTestingHelper.renderComponent();
 
     expect(screen.queryByText('Saving')).toBeNull();
-    expect(screen.getByText('Last saved: Jan. 1')).toBeInTheDocument();
+    expect(screen.getByText('Last saved: Jan 1')).toBeInTheDocument();
```

Comment:
Did this need to change? Tests are failing with
```
          <div
            class="ApplicationFormStatus__SavingStatusContainer-sc-fsr8an-4 foRBhe"
          >
            Last saved: 
            Jan. 1
          </div>
```
Was there a newline before?

Reply:
I'm rolling back this change .. it was made before I changed format strings to match luxon formats.